development/Writing-a-plugin.md

```diff
@@ -36,11 +36,11 @@ class MySuperCoolPlug extends Pluggable{
 
  ```
 
-**A short explanation**: The first 3 Methods `getName`, `getAuthor` and `getDescription` are simple: Just a return a String representing the meta information of your plugin. This inclucde it's name (please make sure to not name two plugins develeoped by yourself the same), your name or pseudonym and a brief topic of the plugins purpose.
+**A short explanation**: The first 3 Methods `getName`, `getAuthor` and `getDescription` are simple: They are just returning a String representing the meta information of your plugin. This includes its name (please make sure to not name two plugins develeoped by yourself the same), your name or pseudonym and a brief topic of the plugins purpose.
```

Comment:
s/the first 3 Methods/The first three methods
s/They are just returning/They just return
"please make sure to not name two plugins devele**o**ped by yourself the same": "please make sure not to name two plugins identically"
s/brief topic/brief summary